papas/reconstruction/src/PFBlock.cpp

```diff
@@ -162,28 +169,34 @@ std::string PFBlock::edgeMatrixString() const {
         if (e1 == e2) {
           out.write("       .");  // diagonal
           break;
-        } else if (edge(e1, e2).distance() < 0)
-          out.write("     ---");  //-ve distance
-        else if (edge(e1, e2).isLinked() == false)
-          out.write("     xxx");  // not linked
-        else {                    // linked and has distance
-          out.write("{:8.4f}", edge(e1, e2).distance());
+        } else if (hasEdge(e1, e2)) {
+          const auto& ed = edge(e1, e2);
```

Comment:
Done. Note I had to quality the edge function to PFBlock::edge to get it to accept  edge as the variable name.. const auto& edge = PFBlock::edge(e1, e2);